lib/assesments/assessment_2/zeeshan/screens/fi_a2_zeeshan_profile_screen.dart

```diff
@@ -0,0 +1,241 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/zeeshan/screens/fi_a2_zeeshan_more_screen.dart';
+import 'package:flutter_interns/assesments/assessment_2/zeeshan/screens/fi_a2_zeeshan_setting_screen.dart';
+import 'package:flutter_interns/assesments/assessment_2/zeeshan/utils/constants.dart';
+import 'package:flutter_interns/assesments/assessment_2/zeeshan/widgets/design/fi_a2_zeeshan_profile_container.dart';
+
+class FIA2ZeeshanProfileScreen extends StatefulWidget {
+  const FIA2ZeeshanProfileScreen({super.key});
+
+  @override
+  State<FIA2ZeeshanProfileScreen> createState() =>
+      _FIA2ZeeshanProfileScreenState();
+}
+
+class _FIA2ZeeshanProfileScreenState extends State<FIA2ZeeshanProfileScreen> {
+  @override
+  void initState() {
+    selectedTab = {'Photos': photoTab(context)};
+    super.initState();
+  }
+
+  Widget photoTab(BuildContext context) => Container(
+      alignment: Alignment.center,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Container(
+            width: 160,
+            decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(20),
+                image: const DecorationImage(
+                    image: AssetImage('assets/fi-a2-zeeshan-photo1.png'),
+                    fit: BoxFit.cover)),
+          ),
+          Column(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              for (int i = 2; i < 5; i++)
+                Container(
+                  height: 95,
+                  width: 160,
+                  decoration: BoxDecoration(
+                      borderRadius: BorderRadius.circular(20),
+                      image: DecorationImage(
+                          image: AssetImage('assets/fi-a2-zeeshan-photo$i.png'),
+                          fit: BoxFit.cover)),
+                ),
+            ],
+          )
+        ],
+      ));
+
+  Widget videoTab(BuildContext context) => Container(
+      alignment: Alignment.center,
+      child: Text(
+        'Video Tab',
+        style: TextStyle(
+          fontFamily: fontFamily,
+          fontWeight: FontWeight.w600,
+          fontSize: 15,
+          color: const Color.fromRGBO(112, 108, 108, 0.5),
+        ),
+      ));
+
+  Widget taggedTab(BuildContext context) => Container(
+      alignment: Alignment.center,
+      child: Text(
+        'Tagged Tab',
+        style: TextStyle(
+          fontFamily: fontFamily,
+          fontWeight: FontWeight.w600,
+          fontSize: 15,
+          color: const Color.fromRGBO(112, 108, 108, 0.5),
+        ),
+      ));
+
+  Map<String, Widget>? selectedTab;
+
+  @override
+  Widget build(BuildContext context) {
+    var screen = MediaQuery.of(context).size;
+
+    Map<String, String> stats = {
+      'Photos': '315',
+      'Followers': '77.5k',
+      'Follows': '500'
+    };
+
+    Map<String, Widget> tabBarItems = {
```

Comment:
Make this widget separately as a class, don't use functions for widgets.

Reply:
## Status
- Issue Resolved